src/pybind/mgr/snap_schedule/fs/schedule_client.py

```diff
@@ -293,6 +309,16 @@ def create_scheduled_snapshot(self,
                     log.info(f'created scheduled snapshot of {path}')
                     log.debug(f'created scheduled snapshot {snap_name}')
                     sched.update_last(time, db)
+                    conn_mgr.dbinfo.reset_path_faults(path)
+                except cephfs.ObjectNotFound:
+                    # maybe path is missing or wrong
+                    self._log_exception('create_scheduled_snapshot')
+                    log.debug(f'path {path} is probably missing or wrong; '
+                              'remember to strip off the mount point path '
+                              'prefix to provide the correct path')
+                    conn_mgr.dbinfo.inc_path_faults(path)
+                    if conn_mgr.dbinfo.get_path_faults(path) >= MAX_PATH_MISSING_FAULTS:
+                        sched.set_inactive(db)
```

Comment:
Oh, deactivating this was never implemented? Let's add a test case for this.

Reply:
Deactivation was **indeed** implemented, but faults weren't counted for allowing a grace period.

Reply:
The issue with the path wasn't highlighted in the logs either.